SmplEditor/MainWindow.xaml.cs

```diff
@@ -93,31 +104,23 @@ private void OnImportPlaylistClicked(object sender, RoutedEventArgs e)
             return availableIndices;
         }
         
-        private void AddToImportedFiles(string[] selectedFileNames, List<int> processedIndices){
-            foreach(int fileIndex in processedIndices){
-                this.importedFileNames.Add(selectedFileNames[fileIndex]);
-            }
-        }
-
         private string GetExtensionFromSafeName(string safeName){
             int extStartIndex = safeName.LastIndexOf(".");
             string extension = safeName.Substring(extStartIndex);
             return extension;
         }
-        
-        private string GetPlaylistNameFromSafeName(string safeName){
-            int extStartIndex = safeName.LastIndexOf(".");
-            string playlistName = safeName.Substring(0, extStartIndex);
-            return playlistName;
-        }
 
-        // private List<T> addNewSongsToLibrary<T,P>(P palylist, List<T> library)
-        private List<Song> matchExistingSongs(Smpl playlist, List<Song> library, List<Song> newSongs){
+        /// <summary> 
+        /// For each track in the imported list, search if the track already exists in the library.<br/>
+        /// If any song was not found in the existing library, convert it to a &lt;Song&gt;.<br/>
```

Comment:
Add a description about how an existing track with a different type is treated.
// If some tracks already exist but are in different type, return the list of them.